libdokan/rekey_file.go

```diff
@@ -24,7 +24,9 @@ func (f *RekeyFile) WriteFile(ctx context.Context, fi *dokan.FileInfo, bs []byte
 	if len(bs) == 0 {
 		return 0, nil
 	}
-	err = f.folder.fs.config.KBFSOps().Rekey(ctx, f.folder.getFolderBranch().Tlf)
+	// TODO: get rid of this and use RequestRekey instead
```

Comment:
The `Rekey` file and all tests want the same thing, so there only needs to be one non-test-specific way of doing that.